crates/examples/attestation/present.rs

```diff
@@ -62,17 +63,63 @@ async fn create_presentation(example_type: &ExampleType) -> Result<(), Box<dyn s
         }
     }
 
-    // Reveal only parts of the response
-    let response = &transcript.responses[0];
-    builder.reveal_recv(&response.without_data())?;
-    for header in &response.headers {
-        builder.reveal_recv(header)?;
-    }
+    // Reveal the following committed parts using one superset range (`R1`):
+    // (1) `request.without_data` (parts excluding target, headers and body).
+    // (2) `request.target`.
+    //
+    // `R1` is constructed via the union of ranges of (1) and (2).
+    builder.reveal_sent(
+        &request
+            .without_data()
+            .union(&request.request.target.to_range_set()),
+    )?;
+
+    /*
+    SIDE NOTE:
+
+    In the code above, we demonstrated how `reveal_sent` can be used on
+    (1) exact range (i.e. each header).
```

Comment:
```suggestion
    (1) exact range (i.e. selected headers).
```

Reply:
it's every header no? just that USER_AGENT and AUTH is without value?

Reply:
I interpret that as skipping a header :-)
You decide

Reply:
good point!